tools/docker/Makefile

```diff
@@ -101,7 +101,13 @@ LINUX_KERNEL_VERSIONS := 4.14.304 \
 	5.18.19 \
 	5.19.17 \
 	6.0.19 \
-	6.1.8
+	6.1.8 \
+	6.2.16 \
+	6.3.13 \
+	6.6.53 \
+	6.8.12 \
+	6.10.12 \
+	6.11.1
```

Comment:
Do we have an official process for how to pick these? Since I had conditional logic for pre 6.3 and post 6.3, I wanted the latest minor version on either side, in addition to some number of kernels between 6.3 and 6.11.

Reply:
The last time I did this I picked the newest patch version for each.

Reply:
Can you check the change in the size of the headers tar with these changes? I believe we were worried about the size increase here. cc @oazizi000

Reply:
This change adds 63MB to the existing header file and brings its total size to 315MB. That's a pretty big addition, so I'd be open to trimming down the added versions. I think we could select one version in between 6.1 and 6.7/6.8 and one from 6.10/6.11 (since #2035 is related to those versions).

Comment updated and will keep it in sync with what we decide as the right set of new headers to include.